llvm/test/CodeGen/AMDGPU/image-sample-waw.mir

```diff
@@ -0,0 +1,23 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
```

Comment:
Maybe call it `waitcnt-sample-waw.mir` by analogy with the existing `waitcnt-vmem-waw.mir`?